src/extensions/modtools.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, bot):
         inumber = int(number)
         if inumber <= 100:
             # In order to delete the command message too, the number of messages to clear is incremented
+            # noinspection PyUnresolvedReferences

```

Comment:
Yeah, that's why I just suppressed it.
I suppose I should just disable that particular inspection and not bother with the in-comment suppressions.